exporter/sentryexporter/sentry_exporter.go

```diff
@@ -57,6 +57,7 @@ func (s *SentryExporter) pushTraceData(_ context.Context, td pdata.Traces) error
 	}
 
 	maybeOrphanSpans := make([]*sentry.Span, 0, td.SpanCount())
+	exceptionEvents := make([]*sentry.Event, 0)
```

Comment:
We can’t estimate the number of span events in anyway? I would encourage atleast setting the capacity to 10 so we minimize allocations.

Reply:
Hmm actually maybe 10 is too much, thoughts on what we should do here?

Reply:
We can start by setting the capacity to 5/10, because currently I don't see any method to pre-calculate number of recorded exception events, and if we do write one such helper method, it'd be an unnecessary computation overhead because of going through all the span events twice.

Reply:
The Go runtime takes care of deciding how to allocate the backing array, we should not need to over-optimize at this point.

In fact, it is idiomatic to not allocate slices with `make` if it may end up empty (thus, no allocation).

```go
var exceptionEvents []*sentry.Event
...
exceptionEvents = append(exceptionEvents, event)  // dynamically allocated, when needed
```

Reply:
@rhcarvalho , acknowledged, in this commit I have adopted your suggestions in the current commit.